infrastructure-playbooks/purge-cluster.yml

```diff
@@ -194,8 +194,8 @@
 
   tasks:
 
-  - name: define regex strings
-    include_vars: ../roles/ceph-osd/defaults/regex.yml
+  - name: define regex vars
+    include_vars: ../roles/ceph-osd/defaults/main.yml
```

Comment:
Instead of using ``include_vars`` I think you could just add the ``ceph-defaults`` role to the playbook.

Above ``tasks`` add:

```
roles:
  - ceph-defaults
```